src/amo/sagas/home.js

```diff
@@ -122,6 +124,16 @@ export function* fetchHomeData({
       },
     };
 
+    const promotedExtensionsParams: SearchParams = {
+      api: state.api,
+      filters: {
+        addonType: ADDON_TYPE_EXTENSION,
+        page_size: String(LANDING_PAGE_PROMOTED_EXTENSION_COUNT),
+        promoted: SPONSORED,
+        sort: SEARCH_SORT_RANDOM,
```

Comment:
no `app` filter?  Or is that added elsewhere?

Reply:
Oh yeah. This shelf is for desktop only, so I guess I should pass `app=firefox`. Thanks.

Reply:
Actually, that *is* added elsewhere, so we don't need it here.